src/fsharp/infos.fs

```diff
@@ -2560,8 +2560,9 @@ let evalFSharpAttribArg e =
         | Const.Zero -> null
         | Const.String s ->  box s
         | _ -> fail()
-    // TODO: typeof<..> in attribute values
-    // TODO: arrays in attribute values
+    | Expr.Op (TOp.Array,_,a,_) -> box [| for i in a -> evalFSharpAttribArg g i |]
+    | TypeOfExpr g ty -> box ty
```

Comment:
This use of "box" is unusual.  I suppose it does make for a minimal diff, which is good, but could you add a comment to evalFSharpAttribArg to indicate the range of objects that can be returned from the function?